sdk/storage/Azure.Storage.Blobs/tests/BlobTestBase.cs

```diff
@@ -161,6 +161,9 @@ public BlobServiceClient GetServiceClient_PremiumBlobAccount_SharedKey()
         public BlobServiceClient GetServiceClient_OauthAccount() =>
             GetServiceClientFromOauthConfig(TestConfigOAuth);
 
+        public BlobServiceClient GetServiceClient_ManagedDisk() =>
+            GetServiceClientFromSharedKeyConfig(TestConfigManagedDisk);
```

Comment:
Can you add a note in https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/CONTRIBUTING.md#configuration explaining what's needed for this?  We'll also need to get this added to the CI setup for these tests to pass.

Reply:
(Should fix)

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/9855